_source/api/logzio-public-api.yml

```diff
@@ -760,8 +764,178 @@ paths:
           schema:
             $ref: '#/definitions/MessageBean'
 
+  # ::::: SHIPPING TOKENS
+  /v1/log-shipping/tokens:
+    post:
+      tags:
+        - Manage log shipping tokens
+      summary: Create a log shipping token
+      description: Creates a log shipping token for this account.
+      operationId: createNewLogShippingToken
+      x-code-samples:
+        - lang: cURL
+          source: >-
+            curl --location --request GET 'https://api.logz.io/v1/account-management/whoami/' \
```

Comment:
This feature needs to be enabled somewhere so you can run test calls with the code samples. Please look into this in the morning.

Reply:
This feature is still not deployed. Waiting for dev.